src/modules/powerrename/lib/PowerRenameRegEx.cpp

```diff
@@ -159,6 +159,38 @@ IFACEMETHODIMP CPowerRenameRegEx::PutFlags(_In_ DWORD flags)
     return S_OK;
 }
 
+IFACEMETHODIMP CPowerRenameRegEx::PutFileTime(_In_ SYSTEMTIME fileTime)
+{
+    union timeunion
+    {
+        FILETIME fileTime;
+        ULARGE_INTEGER ul;
+    };
+
+    timeunion ft1;
+    timeunion ft2;
+
+    SystemTimeToFileTime(&m_fileTime, &ft1.fileTime);
+    SystemTimeToFileTime(&fileTime, &ft2.fileTime);
+
+    if(ft2.ul.QuadPart != ft1.ul.QuadPart)
```

Comment:
You sould use the integrated .clang-format formatting. Did you disable it?

Reply:
No I didn't. What's exactly wrong with it?

Reply:
`if(` should be ` if (`

Reply:
Oh ok, missed that.